src/libs/ReportUtils.ts

```diff
@@ -7483,6 +7483,23 @@ function isExported(reportActions: OnyxEntry<ReportActions>) {
     return Object.values(reportActions).some((action) => ReportActionsUtils.isExportIntegrationAction(action));
 }
 
+function getApprovalChain(policy: OnyxEntry<Policy>, employeeAccountID: number, reportTotal: number): string[] {
+    const approvalChain: string[] = [];
+
+    // If the policy is not on advanced approval mode, we should not use the approval chain even if it exists.
+    if (!PolicyUtils.isControlOnAdvancedApprovalMode(policy)) {
```

Comment:
NAB - Maybe in a follow-up we can change this to only checking if the policy is not on advanced approval mode - it shouldn't matter if it's control or not (currently advanced approval is only available in control, but this "technically" could change at some point, and we'd have to update this)

Reply:
ohhh shoot i used an existing function. i should have asked

Reply:
will fix in a follow up